src/main/java/bio/terra/flight/dataset/ingest/IngestSetupStep.java

```diff
@@ -56,6 +56,24 @@ public IngestSetupStep(DatasetService datasetService, BigQueryPdao bigQueryPdao)
 
     @Override
     public StepResult doStep(FlightContext context) {
+        IngestRequestModel ingestRequestModel = IngestUtils.getIngestRequestModel(context);
+        IngestUtils.GsUrlParts gsParts = IngestUtils.parseBlobUri(ingestRequestModel.getPath());
+
+        // FIXME: This check is preventing us from using wildcard patterns to ingest,
+        //  which is a giant PITA. How can we enable the pattern-matching without losing
+        //  the safety check?
+        //  https://cloud.google.com/bigquery/docs/loading-data-cloud-storage#load-wildcards
```

Comment:
@raaidbroad @thathert the block in question

Reply:
I'll remove the comment and write a short doc, will link it in #dsp-jade when it's ready for 👀